src/classes/HMSSDK.tsx

```diff
@@ -1105,10 +1108,10 @@ export class HMSSDK {
    * @param {*} callback
    * @memberof HMSSDK
    */
-  addEventListener = (
-    action: HMSUpdateListenerActions | HMSPIPListenerActions,
-    callback: any
-  ) => {
+  addEventListener<T extends HMSUpdateListenerActions>(
```

Comment:
@b0iq I think we can keep `addEventListener` method as arrow function

and also can generic type `T` extend `HMSUpdateListenerActions | HMSPIPListenerActions` instead of only `HMSUpdateListenerActions`? if yes, then please add callback types for `HMSPIPListenerActions`in `HMSUpdateListenerCallbacks` interface


```suggestion
  addEventListener = <T extends HMSUpdateListenerActions | HMSPIPListenerActions>(
```